specs/7008-FUND-funding-payment.md

```diff
@@ -0,0 +1,21 @@
+# View my funding payment(s)
+
+- **Must** be able to view my funding payment from the trading page (<a name="7008-FUND-001" href="#7008-FUND-001">7008-FUND-001</a>)
+- **Must** be able to view my funding payment from the portfolio page (<a name="7008-FUND-002" href="#7008-FUND-002">7008-FUND-002</a>)
+
+- **Must** be able to view the following columns (<a name="7008-FUND-003" href="#7008-FUND-003">7008-FUND-003</a>):
+
+  - Market
+  - Amount
+  - Date
+
+- **Must** The columns 'Amount' must change color and have a + or - suffix depending on being positive/negative (<a name="7008-FUND-004" href="#7008-FUND-004">7008-FUND-004</a>)
+- **Must** be able to sort each column by asc and dsc (<a name="7008-FUND-005" href="#7008-FUND-005">7008-FUND-005</a>)
+- **Must** be able to drag and drop column names to re order (<a name="7008-FUND-006" href="#7008-FUND-006">7008-FUND-006</a>)
+- **Must** be able to remove columns by dragging headers out of the table (<a name="7008-FUND-007" href="#7008-FUND-007">7008-FUND-007</a>)
+- **Must** be able to resize the width of the columns (<a name="7008-FUND-008" href="#7008-FUND-008">7008-FUND-008</a>)
+- **Must** retain previous column sizing on refresh (<a name="7008-FUND-009" href="#7008-FUND-009">7008-FUND-009</a>)
+
+# Chart
+
+- **Must** see Funding tab on trading page (<a name="7008-FUND-010" href="#7008-FUND-010">7008-FUND-009</a>)
```

Comment:
Should we add specs for funding rate shown on hover?

Reply:
added